packages/rendering/lib/aws-metrics.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+
+type Metric = {
+    send: () => void,
+};
+
+process.env.AWS_PROFILE = 'frontend';
```

Comment:
What is this used for? Note that environment variables set on `process.env` are not available outside of the Node runtime

Reply:
This makes the node aws client library use the appropriate local aws credentials if they're available, so that it works locally on your machine if you're testing the production server.

Reply:
Okay. Do we always want to set this, or just when we're running the app locally?

Reply:
It should be ok to always set this, as aws credentials work essentially as a series of fallbacks - it should try your local aws profile first, then look for credentials in other standard places. That's how it works in scala too (if it doesn't work in production it should be a quick tweak to force it to look explicitly).

Reply:
Cool, thanks for explaining, sounds good 👍